flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/beans/NtDoubleProperty.java

```diff
@@ -8,7 +8,6 @@
 import java.util.Objects;
 
 public class NtDoubleProperty extends NtPropertyBase implements DoubleProperty {
-
```

Comment:
What are you trying to be declined?
Put those lines back.

Reply:
;-;